provd/proto/accessibility.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package accessibility;
+
+service AccessibilityService {
+    rpc GetHighContract(Empty) returns (GetHighConstrastResponse) {}
+    rpc SetHighContract(SetHighConstrastRequest) returns (Empty) {}
+    rpc GetReducedMotion(Empty) returns (GetReducedMotionResponse) {}
+    rpc SetReducedMotion(SetReducedMotionRequest) returns (Empty) {}
+    rpc GetScreenReader(Empty) returns (GetScreenReaderResponse) {}
+    rpc SetScreenReader(SetScreenReaderRequest) returns (Empty) {}
+    rpc GetLargeText(Empty) returns (GetLargeTextResponse) {}
+    rpc SetLargeText(SetLargeTextRequest) returns (Empty) {}
```

Comment:
In g-i-s today the accessibility service icon is shown (see screenshot below). I don't know what we need to cover in the UI, and will defer to @anasereijo, but I wanted to flag that this doesn't match up with it.

<img width="725" alt="image" src="https://github.com/canonical/ubuntu-desktop-provision/assets/1335525/64e6178a-781d-45f4-b2f7-f4fa8ab3d844">

Reply:
Good points, @tim-hm! I am yet to define what we should display here. On Elio's [design](https://www.figma.com/file/xkI4paNpSjKhAq9cHhrRXx/24.04-Desktop-Provisioning-UX?type=design&node-id=539%3A4975&mode=design&t=UQY1oOqHzVhgjO5l-1) less a11y settings are displayed. However, after looking at what you just shared from the a11y dropdown, I think we can include all those options! We can even then group them to make it look nicer and easier to scan. @matthew-hagemann how costly it is for you to add all options even if there's a chance you will not include them all in the final design?

Reply:
Fairly easy! I will add more and we can pick and choose what to implement when the time comes :slightly_smiling_face: